xbmc/utils/SystemInfo.cpp

```diff
@@ -1243,7 +1243,8 @@ std::string CSysInfo::GetVersion()
 
 std::string CSysInfo::GetBuildDate()
 {
-  return StringUtils::Format("%s", __DATE__);
+  std::string bdate = CCompileInfo::GetBuildDate();
+  return StringUtils::Format("%s", bdate);
```

Comment:
There is no need to call format, GetBuildDate is already a string.
```suggestion
  return CCompileInfo::GetBuildDate();
```